generators/client/templates/angular/package.json.ejs

```diff
@@ -105,14 +105,15 @@
     "rimraf": "2.6.1",
     "source-map": "0.6.1",
     "sourcemap-istanbul-instrumenter-loader": "0.2.0",
-    "style-loader": "0.18.2",
+    "style-loader": "0.20.2",
+    "tapable": "1.0.0-beta.5",
     "to-string-loader": "1.1.5",
     <%_ if (protractorTests) { _%>
     "ts-node": "3.3.0",
     <%_ } _%>
     "tslint": "5.5.0",
     "tslint-config-prettier": "1.6.0",
-    "tslint-loader": "3.5.3",
+    "tslint-loader": "sendilkumarn/tslint-loader",
```

Comment:
Does specifying repo work with npm as well? Can you check

Reply:
as long as the end point has `package.json` and it is `npm` behavior